src/main/java/blackjack/domain/bet/BetAmout.java

```diff
@@ -0,0 +1,24 @@
+package blackjack.domain.bet;
+
+public class BetAmout {
+
+    private static final int MAX = 1_000_000_000;
+    private static final int MIN = 1_000;
+
+    private final int amount;
+
+    public BetAmout(int amount) {
+        validateRange(amount);
+        this.amount = amount;
+    }
+
+    public Profit calculateProfit(double leverage) {
+        return new Profit((int) (amount * leverage));
+    }
+
+    private void validateRange(int amount) {
+        if (amount < MIN || MAX < amount) {
+            throw new IllegalArgumentException("[ERROR] 베팅 금액은 " + MIN + "부터 " + MAX + "이하까지 가능합니다.");
+        }
+    }
```

Comment:
이건 그냥 제가 리뷰하면서 공부한 거 공유드려요~
```suggestion
    private void validateRange(int amount) {
        if (amount < MIN || MAX < amount) {
            throw new IllegalArgumentException(
                    "[ERROR] 베팅 금액은 %d부터 %d이하까지 가능합니다.".formatted(MIN, MAX));
        }
    }
```

저는 보통 위에 같이 쓰는 편이 더 좋을까라고 생각하는데, 그냥 + 연산 쓰는 것보다 성능이 더 안좋다고 하네요;;
가독성 면에서도 리비가 사용한 코드가 더 좋은것 같기도 해서, 제 방법을 제안하려다가 철회했습니다;;
[참고 자료](https://hyune-c.tistory.com/28)